src/conversations.js

```diff
@@ -147,7 +147,7 @@ class Conversation extends EventEmmiter {
     this._clock = setInterval(::this.tick, clockSpeed)
     this._clockSpeed = clockSpeed
     this._processing = false
-    this.messageTypes = messageTypes || ['message', 'text']
+    this.messageTypes = messageTypes || ['message', 'text', 'quick_replies']
```

Comment:
This should be `quick_reply`*

You can actually just do:

```js
bp.convo.start(convo => {
  // etc..
  convo.messageTypes = ['message', 'text', 'quick_reply']
})
```

In the mean time.